examples/openshift-origin/README.md

```diff
@@ -180,7 +180,7 @@ $ sudo -E chown -R ${USER} ${OPENSHIFT_CONFIG}
 Then run the following command to collapse them into a Kubernetes secret.
 
 ```sh
-$ docker run -it --privileged -e="KUBECONFIG=/config/admin.kubeconfig" -v ${OPENSHIFT_CONFIG}:/config openshift/origin cli secrets new openshift-config /config -o json &> examples/openshift-origin/secret.json
+$ docker run -it --privileged -e KUBECONFIG=/kubeconfig -v ~/.kube/config:/kubeconfig -v ${OPENSHIFT_CONFIG}:/config openshift/origin:v1.0.3 cli secrets new openshift-config /config -o json &> ${OPENSHIFT_EXAMPLE}/secret.json
```

Comment:
I'm not familiar with the container so I can't say for sure that any one version will always be present. It would be a more robust solution to determine why latest isn't working and keep latest working, as it will always be present on the image.